test/test_static_join/victim.py

```diff
@@ -0,0 +1,9 @@
+a = "Hello"
```

Comment:
So this is a demo file on which to test? Let's add a docstring on the top.

Reply:
Resolved by getting rid of the whole file – it was just duplicating `CASES` anyway.